src/v/http_test_utils/http_imposter.cc

```diff
@@ -0,0 +1,118 @@
+/*
```

Comment:
nit: maybe put it into `src/v/test_utils`? or `src/v/redpanda/test`? so we keep out test utils in a path that has "test"

Reply:
moved to test_utils